src/app/config.js

```diff
@@ -1,21 +1,36 @@
 /* jshint maxlen:false */
-define(['dojo/has'], function (has) {
+define([
+    'dojo/has',
+    'dojo/request/xhr',
+
+    'esri/config'
+], function (
+    has,
+    xhr,
+
+    esriConfig
+) {
     var apiKey;
-    var ownerName;
+    var quadWord;
+    var ownerName = 'FIBERADMIN';
     if (has('agrc-api-key') === 'prod') {
-        // mapserv.utah.gov
-        // apiKey = 'AGRC-E7FEB434755864';
         // fiberediting.mapserv.utah.gov
         apiKey = 'AGRC-B8627CDB199733';
-        ownerName = 'FIBERADMIN';
+        quadWord = 'kayak-cantina-freddie-cover';
     } else if (has('agrc-api-key') === 'stage') {
         // test.mapserv.utah.gov
         apiKey = 'AGRC-FFCDAD6B933051';
-        ownerName = 'FIBERADMIN';
+        quadWord = 'opera-event-little-pinball';
     } else {
-        // localhost
-        apiKey = 'AGRC-B5D62BD2151902';
-        ownerName = 'DBO';
+        xhr(require.baseUrl + 'secrets.json', {
```

Comment:
we should consider switching this to not be sync since that isn't going to be supported in the future.